crates/cxx-qt-gen/src/generator/rust/method.rs

```diff
@@ -52,6 +59,7 @@ pub fn generate_rust_methods(
                     // CXX ends up generating the source, then we generate the matching header.
                     #[cxx_name = #wrapper_ident_cpp]
                     // Namespace is not needed here
```

Comment:
turns out it is when the namespace is on the `type MyObject` and not on the cxx_qt::bridge